src/fsharp/service/IncrementalBuild.fs

```diff
@@ -1184,17 +1186,24 @@ type IncrementalBuilder(initialState: IncrementalBuilderInitialState, state: Inc
                 return Some(boundModel, state.stampedFileNames.[targetSlot])
         }
 
-    let MaxTimeStampInDependencies stamps =
+    let MaxTimeStampInDependencies stamps fileSlot =
```

Comment:
Passing `-1` is a little weird since we have operated on 0 or above for file slots. But I understand why, as `-1` means look at all the stamps and anything greater than that you want to only look at that many file stamps. So maybe `fileSlot` could be renamed to `fileCount` and add a comment explaining what happens when `-1` is passed.

Reply:
Thanks, will do.